pliers/stimuli/base.py

```diff
@@ -151,8 +152,9 @@ def load_url(source):
             load_url(s)
         elif isdir(s):
             for f in glob(join(s, '*')):
-                load_file(f)
-        elif exists(s):
+                if isfile(f):
```

Comment:
Won't this prevent recursion?

Reply:
I think `if isfile(f) or isdir(f)` should do the trick

Reply:
I don't think it was originally meant to support recursion anyways. `load_file` is a separate function, so I think the original code was just looking for flat files in a folder anyways

Reply:
I think we should make it a recursive call to `load_stims` (and remove the checks)

Reply:
You mean you expect this function to load all files recursively? I'm not sure that's what I'd necessarily expect. 

And Tal, I think I need to specifically call isfile on each file to trigger Datalad, but I could be wrong. I can check.

Reply:
Yea I guess that probably wouldn't be expected, not 100% sure.

Reply:
Yeah, agreed. Okay, let's leave it as is.